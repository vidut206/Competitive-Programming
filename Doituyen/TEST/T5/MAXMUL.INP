3
228078619497377358 448065185460916044 4 2
20494361744438685 591655077361953876 1 8
580924880571097834 586989660026058911 3 9
