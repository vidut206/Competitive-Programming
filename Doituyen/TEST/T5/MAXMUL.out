81
81
64
